pkg/fixtures/alert.go

```diff
@@ -134,6 +134,29 @@ func GetScopedResourceAlert(ID string, clusterID string, namespace string) *stor
 	})
 }
 
+// GetScopedNodeAlert returns a Mock alert attached to a node belonging to the input cluster
+func GetScopedNodeAlert(ID string, clusterID string, nodeID string, nodeName string) *storage.Alert {
+	return copyScopingInfo(&storage.Alert{
```

Comment:
The call to `copyScopingInfo` is not required here, as the Alert_Node type does not contain cluster data.